src/com/lftechnology/java/training/alina/basic/SumOfMultiples.java

```diff
@@ -12,15 +13,15 @@
  */
 public class SumOfMultiples {
 
-	final static Logger LOGGER = Logger.getLogger(SumOfMultiples.class.getName());
+	private static final Logger LOGGER = Logger.getLogger(SumOfMultiples.class.getName());
 
 	public static void main(String[] args) {
 		try {
 			SumOfMultiples sum = new SumOfMultiples();
-			sum.getTotalSum();
-			LOGGER.info("Sum of multiples of 3 & 5 is : " + sum.getTotalSum());
+			Integer getTotalSum = sum.getTotalSum();
```

Comment:
Use primitive data type wherever possible. Primitives types are much faster than reference type due to the fact that stack space is faster than heap space.